src/cropareaest_funcs.py

```diff
@@ -0,0 +1,564 @@
+# Import libraries
+import os
+import pandas as pd
+import numpy as np
+import geopandas as gpd
+import rasterio as rio
+from rasterio.mask import mask
+import cartopy.io.shapereader as shpreader
+from shapely.geometry import box
+from sklearn.metrics import confusion_matrix
+
+
+def return_map_bounds_as_df():
+    "Read the bounding coordinates of the crop mask"
+    root_dir = os.getcwd()
+    prj_raster = [f for f in os.listdir(root_dir) if f.startswith("prj")][0]
```

Comment:
Where is this `prj...` file coming from?

Reply:
Came from GDAL project output https://github.com/nasaharvest/crop-mask/blob/e84b66276e351e2b519f12759707a6d655cbea97/src/cropareaest_funcs.py#L131-L132
But it's removed as the function is not needed anymore.